src/main/java/edu/neu/EDE/ExcelDataExtractor.java

```diff
@@ -43,4 +46,16 @@ private void readFile(File f) throws IOException {
             invalidFiles.add(f.getAbsolutePath());
         }
     }
+
+    private void writeFiles() {
+        WorkbookWriter writer = new WorkbookWriter();
+        writer.setSheetType(DataType.STATISTIC);
+        writer.setColumnType(DataType.STIMULUS);
+        writer.setRowType(DataType.SUBJECT);
+        writer.setData(slideMetricData);
+        writer.write(slideMetricData.getStatistics(), slideMetricData.getStimuli(), slideMetricData.getSubjects(), "slideMetricOutput");
```

Comment:
I think it would be good to set variables or something for the get methods so that we aren't calling all of these other methods within write.